integrations/optimizely/lib/index.js

```diff
@@ -213,6 +214,22 @@ Optimizely.prototype.sendClassicDataToSegment = function(experimentState) {
       variationName: variationNames.join(', ') // eg. 'Variation X' or 'Variation 1, Variation 2'
     };
 
+    // If customProps is provided overide the props with it.
```

Comment:
Is possible/feasible to add this functionality to also the Optimizely X variant (`sendNewDataToSegment`)?

Reply:
Yes, Done.